rescript/plugin_setup.py

```diff
@@ -215,7 +215,12 @@
                 'taxonomy with the highest score (e.g., confidence or '
                 'consensus score). Note that "score" assumes that this score '
                 'is always contained as the second column in a feature '
-                'taxonomy dataframe.',
+                'taxonomy dataframe. "majority" finds the LCA consensus while '
+                'giving preference to majority labels. "super" finds the LCA '
+                'consensus while giving preference to majority labels and '
+                'collapsing substrings into superstrings (e.g., '
+                '"g__Lactobacillus" would be collapsed with '
+                '"g__Lactobacillus_group_A").',
```

Comment:
I found this description confusing. Perhaps this:

>"super" finds the LCA consensus while giving preference to majority labels and collapsing substrings into superstrings. For example, when a more specific taxonomy does not contradict a less specific taxonomy, the more specific one is chosen, if it is the majority ( e.g. "g__Faecalibacterium; s__prausnitzii", will be preferred over "g__Faecalibacterium; s__" ).

Of course, this is assuming I am following the concept of "super" correctly, given both  the `test_merge_taxa_super_lca` and `test_merge_taxa_super_lca_majority` test cases. Otherwise, I am not sure what this help text is actually telling me.